docs/user-guide/configuration/workflow.md

```diff
@@ -10,6 +10,8 @@ toc:
       url: "#defining-workflow-order"
     - title: Advanced Workflow Logic
       url: "#advanced-logic"
+    - title: SCM Branch-specific jobs
```

Comment:
Hmm, to me, this function is like "able to trigger specific branch commit", more than filter.
How about `Specific branch trigger`?

Reply:
`Alternative branches`? `Branches`? I'm still okay with `Branch filtering` too
I know we've been trying to avoid using the word "trigger" in Screwdriver lately, so not sure about `Specific branch trigger`

Reply:
I see, then I will use `Branch filtering`. Thank you.